Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.h

```diff
@@ -136,6 +140,7 @@ class WebFullScreenManager final : public WebCore::EventListener {
 #endif // ENABLE(VIDEO)
 
 #if ENABLE(QUICKLOOK_FULLSCREEN)
+    FullScreenMediaDetails getImageMediaDetails(CheckedPtr<WebCore::RenderImage>, bool);
```

Comment:
Nit: thoughts on making this an enumeration? Makes call sites a little easier to read.

```
enum class IsUpdating : bool { No, Yes };
FullScreenMediaDetails getImageMediaDetails(CheckedPtr<WebCore::RenderImage>, IsUpdating);
```

Reply:
Seems fair!